ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -200,17 +193,36 @@ open class BadgeField: UIView {
 
         // An accessible container must set isAccessibilityElement to false
         isAccessibilityElement = false
+
+        // Update appearance whenever `tokenSet` changes.
+        tokenSet.registerOnUpdate(for: self) { [weak self] in
+            self?.updateAppearance()
+        }
     }
 
     @objc private func themeDidChange(_ notification: Notification) {
```

Comment:
Also shouldn't this function be removed since we have `registerOnUpdate`?

Reply:
That was my original thought but I noticed is was still in `Separator` so I wasn't sure if behind the scenes it was still necessary.